packages/grafana-data/src/transformations/matchers/fieldTypeMatcher.ts

```diff
@@ -3,7 +3,7 @@ import { FieldMatcherID } from './ids';
 import { FieldMatcherInfo } from '../../types/transformations';
 
 // General Field matcher
-const fieldTypeMacher: FieldMatcherInfo<FieldType> = {
+const fieldTypeMatcher: FieldMatcherInfo<FieldType> = {
```

Comment:
Renamed these, I hope that's ok for everyone.

Reply:
dooh -- thank you.  You see my dyslexia in full effect here.

Reply:
even update this in its own PR (and commit today!) if you want